src/imagery/i.eodag/i.eodag.py

```diff
@@ -146,6 +155,13 @@
 # % guisection: Sort
 # %end
 
+# %option
+# % key: query
+# % label: Extra searching parameters to use in query
+# % description: Note: Make sure to use provided options when possible, otherwise the values mignt not be recognized
```

Comment:
What is the message here? I do not understand

Reply:
What I am trying to convey is that if you are able to use one of the provided i.eodag options e.g. `start`, `end`, `producttype`, `provider`, etc... Then you should use them instead of passing them through the query option, because they might cause unexpected problems.

Shall I just type this, if it is more clear? or is it too wordy?

Reply:
If I understand right, this option is to use searching parameters not provided for by the other filter parameters? Perhaps turning it around:

"Query using extra searching (filter?) parameters not provided for by the provided filter options"

Would it be possible to check if the user uses query parameters that are already available as parameter option, and issue a warning if the user uses these parameters?

Reply:
This dhould be made clear in the manual...